src/models/FormInterface.ts

```diff
@@ -58,6 +59,7 @@ export interface FieldsDefinitions {
   autoFocus?: any;
   inputMode?: any;
   refs?: any;
+  classes?: object;
```

Comment:
just added this. missed it the first time around. problem made itself evident when i was testing out the docs code and did a `new Form` instantiation with unified fields.